core/sched.c

```diff
@@ -221,3 +202,36 @@ NORETURN void sched_task_exit(void)
     sched_active_thread = NULL;
     cpu_switch_context_exit();
 }
+
+#ifdef MODULE_SCHED_CB
+void sched_register_cb(void (*callback)(uint32_t, uint32_t))
+{
+    sched_cb = callback;
+}
+#endif
+
+#ifdef MODULE_SCHEDSTATISTICS
+void sched_statistics_cb(uint32_t active_thread, uint32_t next_thread) {
+
+    uint32_t now = xtimer_now().ticks32;
+
+    if((thread_getstatus(active_thread) != STATUS_STOPPED) && \
```

Comment:
With this, condition, it currently seams that stopping a thread means its `runtime_ticks` will not be updated.
However it looks like `ps` would still display it.

You could just rely on `pid_is_valid(active_thread)` I guess.

Reply:
yep